constant-pool.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "type.h"
+
+typedef enum {
+    CONSTANT_Utf8 = 1,
+    CONSTANT_Integer = 3,
+    CONSTANT_Long = 5,
+    CONSTANT_Class = 7,
+    CONSTANT_FieldRef = 9,
+    CONSTANT_MethodRef = 10,
+    CONSTANT_NameAndType = 12
```

Comment:
Add a comma (`,`) at the end of `CONSTANT_NameAndType = 12`, so that `clang-format` can indent better.